packages/components/pie-textarea/src/index.ts

```diff
@@ -133,16 +128,6 @@ export class PieTextarea extends FormControlMixin(RtlMixin(LitElement)) implemen
         this._throttledResize();
     }
 
-    private restrictInputLength () {
```

Comment:
If we decide to keep the current `label` prop approach, I think we should consider leveraging max-length as browsers will do that natively.